qoqo/src/devices/mod.rs

```diff
@@ -82,10 +82,10 @@ impl ChainWithEnvironmentDevice for ChainWithEnvironmentCapsule {
                 .internal
                 .call_method0(py, "__environment_chains")
                 .expect("Internal error `environment_chains` on python device failed.");
-            let chains_with_environment = chains_with_environment
-                  .extract::<Vec<(Vec<usize>, HashMap<usize, Vec<usize>>)>>(py)
-                  .expect("Internal error `environment_chains` on python device does not return valid desctiption.");
+
```

Comment:
This change is not reflected in the CHANGELOG, could you add this?

Reply:
There's no functional change, it's just clippy that suggested to do a one-liner instead of instantiating something that is immediately returned.

Reply:
Ah fair